### PR TITLE
Migrate bringup post-submit targets to cocoon scheduler

### PR DESCRIPTION
**Pull request by keyonghan**
Part of https://github.com/flutter/flutter/issues/92300

### CODE REVIEW

File: .ci.yaml [hunk 1]
```diff
@@ -1374,7 +1374,6 @@ targets:
       tags: >
         ["devicelab","android","linux"]
       task_name: clipper_cache_perf__e2e_summary
-    scheduler: luci
```
Comment (CaseyHillers) [inline .ci.yaml:1377]: Is backfilling fixed? I remember we had to revert back as the teams still want access to benchmarks on every commit

https://github.com/flutter/flutter/issues/100793

**Review by ricardoamador — APPROVED** [review]
LGTM

File: .ci.yaml (hunk 1 above)
Comment (keyonghan) [reply .ci.yaml:1377]: Fixed.

**Review by CaseyHillers — APPROVED** [review]
flutter-lgtm

Thank you for fixing the backfilling issues! I'm happy to see this migration wrapping up

**Pull request merged**